src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -146,24 +142,33 @@ sudo apt-get install cmake pkg-config libssl-dev git clang libclang-dev protobuf
 
 <tab-item title="Mainnet">
   ```bash
-  git clone https://github.com/centrifuge/centrifuge-chain.git
-  git checkout [latest_release]
+  git clone https://github.com/centrifuge/centrifuge-chain
   cd centrifuge-chain
-  RUST_TOOLCHAIN=nightly-2022-11-14 ./scripts/install_toolchain.sh
+  latestTag=$(curl -s https://api.github.com/repos/centrifuge/centrifuge-chain/releases/latest | grep '.tag_name'|cut -d\" -f4)
```

Comment:
latest might not really work if it's not ready for mainnet yet. I think validators should manually check the releases page and get the correct tag